README.md

```diff
@@ -1,15 +1,32 @@
 # NF1 Cell Painting Data 
 
-In this repository, we perform the image-based analysis and some analysis of the morphology data.
+In this repository, we generate image analysis and image-based profiling pipelines to extract and format single-cell morphological profiles.
 
-We train a machine learning model to predict NF1 genotype within a separate repository called: [NF1_SchwannCell_data_analysis](https://github.com/WayScience/NF1_SchwannCell_data_analysis).
-Please visit the above repository for further information on the generation and validation of the logistic regression model.
+We train a machine learning model to predict NF1 genotype, evaluate, and generate figures within a separate repository called: [NF1_SchwannCell_data_analysis](https://github.com/WayScience/NF1_SchwannCell_data_analysis).
+Please visit the above repository for further information on the generation, validation, and figures from this model for the manuscript.
 
-**Note:** All metadata files are located in the [download data module](./0.download_data/). All larger files, including `SQLite` outputs from CellProfiler and `parquet` processed data file from pycytominer, will need to be downloaded using git LFS after the repo is cloned.
+## Goal
```

Comment:
Please reformat to one sentence per line in this subsection

Reply:
👍 Thanks for catching that! It is fixed!